newdle/client/src/components/NewdlesParticipating.js

```diff
@@ -0,0 +1,108 @@
+import client from '../client';
+import {usePageTitle} from '../util/hooks';
+import styles from './MyNewdles.module.scss';
+
+export default function NewdlesParticipating() {
+  const [participations, loading] = client.useBackend(() => client.getNewdlesParticipating(), []);
+  usePageTitle("Newdles I'm participating in");
+
+  let content;
+  if (loading || participations === null) {
+    content = (
+      <>
+        <Placeholder className={styles.newdle} />
+        <Placeholder className={styles.newdle} />
+        <Placeholder className={styles.newdle} />
+      </>
+    );
+  } else if (participations.length === 0) {
+    content = (
+      <div className={styles['no-newdle-container']}>
+        {/* eslint-disable-next-line jsx-a11y/accessible-emoji */}
+        <h2>You are not part of any newdles yet.</h2>
+      </div>
+    );
+  } else {
+    participations
+      .sort((a, b) => new Date(a.newdle.final_dt) - new Date(b.newdle.final_dt))
+      .sort((a, b) => !!a.newdle.final_dt - !!b.newdle.final_dt)
+      .sort((a, b) => Object.keys(a.answers).length - Object.keys(b.answers).length)
+      .sort(
+        (a, b) =>
+          (a.newdle.final_dt && new Date(a.newdle.final_dt) < new Date()) -
+          (b.newdle.final_dt && new Date(b.newdle.final_dt) < new Date())
+      );
+    content = participations.map(participation => (
+      <NewdleParticipation
+        key={participation.newdle.id}
+        newdle={participation.newdle}
+        answers={participation.answers}
+        participant_code={participation.code}
+      />
+    ));
+  }
+
+  return (
+    <Container text className={styles.newdles}>
+      {content}
+    </Container>
+  );
+}
+
+function NewdleParticipation({
+  newdle: {code, title, duration, final_dt: finalDT, timezone, timeslots},
+  answers,
+  participant_code,
+}) {
+  const history = useHistory();
+  const startTime = finalDT && serializeDate(finalDT, 'HH:mm');
+  const endTime = finalDT && serializeDate(toMoment(finalDT).add(duration, 'm'), 'HH:mm');
+  const url = flask`newdle`({code, participant_code});
+  const slotsChosen = timeslots.filter(timeslot =>
+    ['available', 'ifneedbe', 'unavailable'].includes(answers[timeslot])
+  );
+
+  return (
+    <div className={styles.newdle} onClick={() => history.push(url)}>
+      <h3 className={styles.title}>
+        <a href={url} onClick={evt => evt.preventDefault()}>
+          {title}
+        </a>
+        <Label color={finalDT ? 'blue' : 'green'} size="tiny" className={styles.state}>
+          {finalDT ? 'finished' : 'ongoing'}
+        </Label>
+      </h3>
+      {finalDT && (
+        <div className={styles.info}>
+          <span>
+            <Icon name="calendar alternate outline" />
+            <label>{serializeDate(finalDT, 'dddd, D MMMM')}</label>
+          </span>
+          <span>
+            <Icon name="clock outline" />
+            <label>
+              {`${startTime} - ${endTime}`} ({timezone})
```

Comment:
`{startTime} - {endtime} ({timezone)`?

Reply:
This was copied from `MyNewdles.js`. Should I keep it, or apply the same style in both?

Reply:
i'd change it in both places unless it results in some ugly way of wrapping after running prettier